include/jemalloc/internal/spin_delay_arm.h

```diff
@@ -0,0 +1,30 @@
+#include "jemalloc/internal/jemalloc_preamble.h"
+#include <sys/auxv.h>
```

Comment:
This will have to be guarded:
https://app.travis-ci.com/github/jemalloc/jemalloc/jobs/632564580#L784